### DISCUSSION TITLE
签到模板失效后有没有方法直接更新模板，不用删除旧的签到任务再新建签到

### DISCUSSION THREAD

**Discussion by yzliufujin** [qd-today/qd#352] [Q&A]
现在我发现某个签到失效后，一般都是先删除原来的签到任务和模板，订阅更新后的签到模板后再新建签到任务，比较繁琐。
请问有没有方法不删除原来的签到任务，直接更新模板呢？

**Comment by a76yyyy** [comment]
Todo

**Reply by a76yyyy** [reply]
现在源码里有一套方式，但不好用，后续会重新设计
